docs/operations/downgrade-ondat-2.7-to-2.6.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Downgrade Ondat from 2.7.0 to 2.6.0"
+linkTitle: Downgrade Ondat from 2.7.0 to 2.6.0
+weight: 150
+---
+
+# Overview
+
+This guide will walk you through how to downgrade from Ondat v2.7.0 to v2.6.0. You can use this procedure should you decide that you need to roll back after upgrading to 2.7.0.
+
+As part of the 2.7.0 release we are implementing a new design for mapping your Kubernetes volumes to the underlying data storage containers on disk. This will make operation and debugging of any issues much easier, as from 2.7.0 the storage files on disk will have the same name (UUID) as the Physical volumes (PV) in Kubernetes.
+
+As part of the upgrade to 2.7.0, there will be a one time step to rename these blob files on disk. This will not change the data in any way, but will update the names of the files to align with the K8s naming convention.
+
+As part of any operational upgrade plans, we want to provide simple steps should you need to roll back in case of issues. The procedure below has been validated, however please do raise a proactive case [here](/docs/support/) ahead of any upgrades and work with the customer success teams as part of any upgrade process.
+For those curious, in the past Ondat supported other Container Orchestrators (CO) and therefore used an internal UUID reference for these blob files. With the focus on only K8s now, we are removing this abstraction layer and the naming will reflect the K8s objects.
+
+# Prerequisites
+
+> ⚠️ Make sure all workloads using Ondat volumes are scaled down to zero.
+
+> ⚠️ Recommended: While the procedure is safe, it is recommended that a backup of important stateful application is kept before performing the downgrade.
+
+> ⚠️ Update the CLI_TOOL variable if you do not have access to kubectl.
+
+> 💡 The tool is idempotent so in the case of interruption it can be safely run multiple times.
+
+# Procedure
+
+## Step 1 - Uninstall Ondat v2.7.0, as if you are starting an upgrade
+
+1. Delete your storageoscluster CR
+1. Delete your storageos operator deployment
+1. Make sure to leave your Ondat etcd alone
+
+## Step 2 - Run our downgrade script
+
+1. Pipe the script for your information by running the following command:
+
+    ```
+    curl -s https://docs.ondat.io/v2.7/sh/downgrade-db-2-7-to-2-6.sh
+    ```
+
+1. Run the script below:
+
+    ```
+    curl https://docs.ondat.io/v2.7/sh/downgrade-db-2-7-to-2-6.sh | bash
```

Comment:
This should be:
```
./downgrade-db-2-7-to-2-6.sh
```
since the script should already have been downloaded and edited.